tests/inverted-index-test.js

```diff
@@ -95,30 +95,29 @@ describe('Inverted Index Suite', () => {
     });
     it('Ensures index returns the correct results when searched',
       () => {
-        expect(newIndex.searchIndex('inquiry', 'books')).toEqual({
+        expect(newIndex.searchIndex('books', 'inquiry')).toEqual({
           'inquiry': [0]
         });
-        expect(newIndex.searchIndex('this', 'books')).toEqual({
+        expect(newIndex.searchIndex('books', 'this')).toEqual({
           'this': [0, 1]
         });
-        expect(newIndex.searchIndex('inertia', 'books')).toEqual({
-          'inertia': 'Not Found'
+        expect(newIndex.searchIndex('books', 'inertial')).toEqual({
+          'inertial': 'Not Found'
         });
-        expect(newIndex.searchIndex('third first nations', 'books')).toEqual({
-          'third':[1],
+        expect(newIndex.searchIndex('books', 'third first nations')).toEqual({
+          'third': [1],
           'first': [1],
           'nations': [0]
         });
-    });
-    it('Ensures it searches all indexed files if a filename/key is not passed',
-      () => {
-        expect(newIndex.searchIndex('inquiry')).toEqual([{ 'inquiry': [0] }]);
-        expect(newIndex.searchIndex('this')).toEqual([{
-          'this': [0, 1]
-        }]);
-        expect(newIndex.searchIndex('inertia')).toEqual([{
-          'inertia': 'Not Found'
-        }]);
       });
+    /* it('Ensures it searches all indexed files if a filename/key is not passed', () => {
+      expect(newIndex.searchIndex('inquiry')).toEqual([{ inquiry: [0] }]);
+      expect(newIndex.searchIndex('this')).toEqual([{
+        this: [0, 1]
+      }]);
+      expect(newIndex.searchIndex('inertia')).toEqual([{
+        inertia: 'Not Found'
+      }]);
+    });*/
```

Comment:
Nice work, but I think it's necessary that you remove commented codes from your files.